go.mod

```diff
@@ -4,25 +4,21 @@ go 1.13
 
 require (
 	github.com/Knetic/govaluate v3.0.1-0.20171022003610-9aa49832a739+incompatible
-	github.com/apex/log v1.3.0 // indirect
 	github.com/aws/aws-sdk-go v1.31.3
 	github.com/dustin/go-humanize v1.0.0
 	github.com/gorilla/handlers v1.4.2
 	github.com/gorilla/mux v1.7.4
-	github.com/jmoiron/sqlx v1.2.0 // indirect
-	github.com/lib/pq v1.5.2 // indirect
+	github.com/kr/pretty v0.2.0 // indirect
```

Comment:
what are all these packages that you added/removed?

Reply:
noting special go mod tidy (cleans up unnecessary deps)